welcome/about-kueue.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="about-kueue"]
+= About {product-title}
+:context: about-kueue
+
+toc::[]
+
+{product-title} is a Kubernetes-native system that manages quotas and how jobs consume them. Kueue decides when a job waits, when a job is admitted to start by creating pods, and when a job should be _preempted_ (active pods should be deleted).
```

Comment:
So Kueue has a concept of an integration (Job, PyTorchJob, JobSet, Pod, Deployment, StatefulSet).

In reality, Kueue can decide when to admit these workloads.

The docs mention Jobs so I know it can be confusing as we expalin the other integrations. It is something to keep in mind..

Kueue takes the approach that "Job" is a general workload and it means any kind of Kueue workload. I'm not sure if that is what we mean here.

Reply:
probably better to use "workload" in that case, since Job generally means something so specific :)